src/dbnode/storage/namespace_test.go

```diff
@@ -1188,6 +1188,40 @@ func TestNamespaceIndexQuery(t *testing.T) {
 	assert.Equal(t, "root", spans[1].OperationName)
 }
 
+func TestNamespaceIndexWideQuery(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	idx := NewMockNamespaceIndex(ctrl)
+	idx.EXPECT().BootstrapsDone().Return(uint(1))
+
+	ns, closer := newTestNamespaceWithIndex(t, idx)
```

Comment:
Add a test for namespace without index, to make sure we avoid NPE and return a proper error.